src/Files.Uwp/App.xaml.cs.old

```diff
@@ -0,0 +1,758 @@
+using CommunityToolkit.Mvvm.DependencyInjection;
```

Comment:
What's this file for?

Reply:
Can be removed. I think we ported everything from it except prelaunch.

Reply:
I noticed that continue where you left off is broken, is that part of the launch arguments?

Reply:
No, open tabs are saved to LocalSettings in App.cs/SaveSessionTabs() and restores in MainPageViewModel/OnNavigatedTo() (or they are supposed to). The code for it is still in place.

Reply:
Continue where you left off works fine for me with latest commits, could you check?

Reply:
Works for me as well 👍